components/grouper/src/main/java/org/datacleaner/components/group/GrouperTransformer.java

```diff
@@ -132,7 +132,8 @@ public void init() {
         outputDataStreamBuilder.withColumn("row_count", ColumnType.INTEGER);
         for (int i = 0; i < aggregatedValues.length; i++) {
             final InputColumn<?> inputColumn = aggregatedValues[i];
-            final AggregationType aggregationType = aggregationTypes[i];
+            final AggregationType aggregationType = (aggregationTypes.length <= i ? AggregationType.CREATE_LIST
+                    : aggregationTypes[i]);
```

Comment:
During updates of the graph this method will be called multiple times - also before the property widget has been able to provide an 'aggregationTypes' value which matches the size of the 'aggregatedValues' value.